src/multi/uvarint.cpp

```diff
@@ -76,6 +76,10 @@ namespace libp2p::multi {
     return !(*this == r);
   }
 
+  bool UVarint::operator<(const UVarint &r) const {
```

Comment:
You can use https://www.boost.org/doc/libs/1_65_0/libs/utility/operators.htm to add all operators without boiler-plate.